### PR TITLE
PMM-4544 Add Node params in Management API.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-415/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-415/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-415/

File: api/managementpb/mysql.proto [hunk 1]
```diff
@@ -24,17 +24,17 @@ import "github.com/mwitkow/go-proto-validators/validator.proto";
...
   // Node identifier on which a service is been running. Required.
-  string node_id = 1 [
-    (validator.field) = {
-      string_not_empty: true
-    }
-  ];
+  // Use only one of these paramse (node_id, node_name or register_node)
+  string node_id = 1;
+  string node_name = 21;
+  RegisterNodeRequest register_node = 22;
```
Comment (AlekSi) [inline api/managementpb/mysql.proto]: All `XXXRequest` messages must be independent and not try to share common properties for future compatibility. Additionally, `AddMySQLRequest` and `RegisterNodeRequest` already share a field: `address`.

We have to copy&paste fields there.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-415/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-415/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-415/

File: api/managementpb/mysql.proto [hunk 2]
```diff
@@ -83,6 +83,35 @@ message AddMySQLRequest {
   bool skip_connection_check = 30;
 }
 
+// AddNodeRequest adds new node to inventory while adding new service.
+message AddNodeRequest {
```
Comment (AlekSi) [inline api/managementpb/mysql.proto]: `XXXRequest` and `XXXResponse` would be used only for RPC requests and responses. `AddNodeParams`?

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-415/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-415/

**Pull request merged**